man/systemd.netdev.xml

```diff
@@ -983,6 +986,100 @@
     as the <literal>[Tun]</literal> section.</para>
   </refsect1>
 
+  <refsect1>
+    <title>[WireGuard] Section Options</title>
+
+    <para>The <literal>[WireGuard]</literal> section accepts the following
+    key:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>PrivateKey=</varname></term>
+        <listitem>
+          <para>The private key for the interface. It can be generated
+            using the <command>wg genkey</command> command. This option is
```

Comment:
hmm, the first time we mention the `wg` tool we should add a proper man page reference, i.e. something like `<citerefentry><refentrytitle>wg</refentrytitle><manvolnum>somenumber</manvolnum></citerefentry>`

Reply:
`wg(8)`.